src/main/java/org/spongepowered/common/event/SpongeEventManager.java

```diff
@@ -411,16 +414,18 @@ private boolean post(Event event, List<RegisteredListener<?>> handlers) {
             try (CauseStackManager.StackFrame frame = Sponge.getCauseStackManager().pushCauseFrame();
                  final PhaseContext<?> context = PluginPhase.Listener.GENERAL_LISTENER.createPhaseContext()
```

Comment:
extra leading space?

Reply:
That's the proper indentation since it's a multiple try with resources.